### ISSUE TITLE
rpnzl not working

### ISSUE THREAD

**Issue by chikno** [rpnzl/jquery-lazyjson#3]
Thank you for this plugin.
But i tried all the demos in your website,  but not working. it only returns  : 
{{data.first_name}} {{data.last_name}}.
ID: {{id}}

Any help would be appreciable.
Thank you

**Comment by chikno** [comment]
Please Help

**Comment by rpnzl** [comment]
I'll need more context on how you're using the plugin to help - can you provide code samples and an example API response?

**Comment by chikno** [comment]
thank you for your reply.

I tried to do as the same Démo examples in your website in the quick start Section .

Index.php : 
<!DOCTYPE html>

   
      
          
         <html class="no-js">
            
            <head>
               <meta charset="utf-8">
               <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
               <title></title>
               <meta name="description" content="">
               <meta name="viewport" content="width=device-width, initial-scale=1">
               <link rel="stylesheet" href="css/bootstrap.min.css">
               <style>
                  body {
                  padding-top: 50px;
                  padding-bottom: 20px;
                  }
               </style>
               <link rel="stylesheet" href="css/bootstrap-theme.min.css">
               <link rel="stylesheet" href="css/main.css">

<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.8.1/jquery.min.js"></script>
<script src="js/lazyJson.js"></script>

<script src="js/main.js"></script>  
            </head>
            <body>
                  <div>
         
               <div class="navbar navbar-inverse navbar-fixed-top" role="navigation">
                  <div class="container">
                     <div class="navbar-header">

                        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
                        <span class="sr-only">Toggle navigation</span>
                        <span class="icon-bar"></span>
                        <span class="icon-bar"></span>
                        <span class="icon-bar"></span>
                        </button>
                        <a class="navbar-brand" href="#">Projet Pige</a>
                     </div>
                     <div class="navbar-collapse collapse">
                        <ul class="nav navbar-nav navbar-right">
                           <li><a href="#">Menu 1</a></li>
                           <li><a href="#">Menu 2</a></li>
                           <li><a href="#">Menu 3</a></li>
                        </ul>
                     </div>
                     
                  </div>
               </div>
          
          
<div id="lazyjson">

    
    
    <div id="template-lazyjson">
        <h3>{{data.first_name}} {{data.last_name}}</h3>
        <p>ID: {{id}}</p>
    </div>
    

</div>

        </div>
                     



            </body>
         </html>

images.json : 

[
    {
        id: 0,
        data: {
            first_name: "Allison",
            last_name: "Baggins"
        }
    },
    {
        id: 1,
        data: {
            first_name: "Arthur",
            last_name: "Baggins"
        }
    },
]

main.js : 
$( 'div#lazyjson' ).lazyjson({
    api: {
        uri: 'http://localhost/pige2/js/images.json'
    }
});

**Comment by chikno** [comment]
thank you for your reply.

I tried to do as the same Démo examples in your website in the quick start Section .
here my files attached.
![image](https://cloud.githubusercontent.com/assets/3385648/6552016/e050568e-c638-11e4-89f9-efe64875024c.png)
![image](https://cloud.githubusercontent.com/assets/3385648/6552020/e59952e4-c638-11e4-89c6-f314916e863f.png)
![image](https://cloud.githubusercontent.com/assets/3385648/6552023/e958b64a-c638-11e4-8cf7-2bfcabe94adc.png)

**Comment by chikno** [comment]
thank you for your reply.

I tried to do as the same Démo examples in your website in the quick start Section .
here my code files attached.

![mainjs](https://cloud.githubusercontent.com/assets/3385648/6552106/c2b05ea2-c639-11e4-92f2-52836de56c40.png)
![datajson](https://cloud.githubusercontent.com/assets/3385648/6552112/de9450a6-c639-11e4-88a0-8105031009b5.png)

![indexphp](https://cloud.githubusercontent.com/assets/3385648/6552108/c9cc74a0-c639-11e4-91ee-1c1d0c73e00e.png)

**Comment by rpnzl** [comment]
Hey, it looks like you're serving up invalid JSON from your images.json -- the example I provided isn't valid so a copy/paste won't work for you. You need quotations and to avoid trailing commas, etc. e.g.

``` json
[
  {
    "id": 0,
    "data": {
      "first_name": "Allison",
      "last_name": "Baggins"
    }
  },
  {
    "id": 1,
    "data": {
      "first_name": "Arthur",
      "last_name": "Baggins"
    }
  }
]
```
